util/keycloak_client.py

```diff
@@ -110,3 +134,34 @@ def create_project(self, charge_code, pi_username):
         keycloakproject.create(charge_code)
 
         self.update_membership(charge_code, pi_username, 'add')
+
+    def create_user(self, username, first_name=None, last_name=None, email=None,
+                    affiliation_title=None, affiliation_department=None,
+                    affiliation_institution=None, country=None, citizenship=None, join_date=None):
+        try:
+            keycloakuser = self._users_admin()
+            keycloakuser.create(
```

Comment:
Does this not return the user? If it returns the user (with ID), you don't need the following lookup.

Reply:
it returns an empty string.